src/services/create-project.service.js

```diff
@@ -52,7 +52,7 @@ export default (
     return;
   }
 
-  const parentPath = defaultParentPath;
+  const parentPath = getDefaultProjectHomePath();
```

Comment:
As I mentioned earlier, now that the path lives in Redux state, we shouldn't access it in the same way as before. While your PR works great, I've found in my experience that React/Redux applications are much easier to understand when the following premise is true:

- Redux state is only accessed from React components, through `mapStateToProps` methods

In order to comply with this premise, we need to fetch the `projectHomePath` from within a component, and pass it into this method.

So, this method should now take a new argument, `projectHomePath`:

```
export default (
  { projectName, projectType, projectIcon }: ProjectInfo,
  projectHomePath: string,
  onStatusUpdate: (update: string) => void,
  onError: (err: string) => void,
  onComplete: (packageJson: any) => void
) => {
```

(Within this method, you can just rename `parentPath` to use the argument `projectHomePath`.)

This `create-project.service` method is called in `BuildPane`, which is used in `CreateNewProjectWizard`. Inside `CreateNewProjectWizard`, you can update `mapStateToProps` to use the new selector:

```js
const mapStateToProps = state => ({
  projects: getById(state),
  projectHomePath: getProjectHomePath
  isVisible: state.modal === 'new-project-wizard',
});
```

Then, it's just a matter of passing that prop down to `BuildPane`, and then invoking the `createProject` method with that prop.

I hope this makes sense! Happy to clarify if something is confusing.